### DISCUSSION TITLE
Indicator for map / theme updates

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#685] [Q&A] [answered]
In the past there was an update message when starting Cruiser. What about a red dot in the map tile indicating that map/theme/POI updates are available? The message was useful as well. What do you think?

**Comment by devemux86** [comment, marked as answer]
@kds112 Thanks for the suggestion.

Offline maps are not updated very often and are not that important to update.
We can always check for updates in the offline maps screen.

Offline routing data are updated daily, these are important to update regularly.
